src/particles/transformation/ToFixedT.H

```diff
@@ -79,11 +79,11 @@ namespace transformation
 
             // transform position and momentum (from fixed s to fixed t)
 
-            p.pos(0) = x + px*t/(m_ptd+pt);
+            p.pos(RealAoS::x) = x + px*t/(m_ptd+pt);
             // px = px;
-            p.pos(1) = y + py*t/(m_ptd+pt);
+            p.pos(RealAoS::y) = y + py*t/(m_ptd+pt);
             // py = py;
-            p.pos(2) = pz*t/(m_ptd+pt);  // This now represents z.
+            p.pos(RealAoS::t) = pz*t/(m_ptd+pt);  // This now represents z.
             pt = pz - pzd;                     // This now represents pz.
```

Comment:
```suggestion
            pz = pzf - pzdf;                     // This now represents pz.
```

Reply:
It remains to make sure "pzf" and "pzdf" are defined here, analogously to "ptf" and "ptdf" in ToFixedS.H.

Reply:
What does the `f` stand for?

Reply:
Updated to be in sync: added `f` suffixes.